nvflare/lighter/impl/cert.py

```diff
@@ -87,6 +87,17 @@ def _build_write_cert_pair(self, participant, base_name, ctx):
             pri_key = serialization.load_pem_private_key(
                 self.persistent_state[subject]["pri_key"].encode("ascii"), password=None, backend=default_backend()
             )
+            if participant.type == "admin":
```

Comment:
What's the reason we only do this for the "admin" type?

Reply:
Only admin certs have unstructured name, which contains the 'role' information, such as project_admin.